### PR TITLE
Add Codeowners

### PR DESCRIPTION
**Pull request by vatsalyagoel**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Add CODEOWNERS

### :arrow_heading_down: What is the current behavior?
Core maintainers have to review everything

### :new: What is the new behavior (if this is a feature change)?
Let people own different platforms

### :memo: Links to relevant issues/docs
#3034 

### :thinking: Checklist before submitting

- [x] Rebased onto current develop

### CODE REVIEW

**Comment by nickrandolph** [comment]
@Cheesebaron any idea why the build is failing for this PR - I've tried forcing the build again but it's still failing.

**Comment by Cheesebaron** [comment]
@nickrandolph I think it is a race condition when running tests using IoC. Will have a look later today.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# standard @username or @org/team-name format. You can also refer to a user by an
+# email address that has been added to their GitHub account, for example user@example.com
+
+*                                          @mvvmcross/core-team
+
+.github/*                                  @mvvmcross/maintainers
+version.json                               @mvvmcross/maintainers
+ContentFiles                               @mvvmcross/maintainers
+
+build.*                                    @mvvmcross/devops-team
```
Comment (Cheesebaron) [inline .github/CODEOWNERS]: Just call this devops

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# A CODEOWNERS file uses a pattern that follows the same rules used in gitignore files.
+# The pattern is followed by one or more GitHub usernames or team names using the
+# standard @username or @org/team-name format. You can also refer to a user by an
+# email address that has been added to their GitHub account, for example user@example.com
+
+*                                          @mvvmcross/core-team
```
Comment (Cheesebaron) [inline .github/CODEOWNERS]: rename to core

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+build.*                                    @mvvmcross/devops-team
+signclient.json                            @mvvmcross/devops-team
+*.ps1                                      @mvvmcross/devops-team
+NuGet.*                                    @mvvmcross/devops-team
+appveyor.yml                               @mvvmcross/devops-team
+tools/*                                    @mvvmcross/devops-team
+MvvmCross.Analyzers/*                      @mvvmcross/devops-team
+
+Projects/*                                 @mvvmcross/learning-team
```
Comment (Cheesebaron) [inline .github/CODEOWNERS]: docs

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+snippets/*                                 @mvvmcross/learning-team
+*.md                                       @mvvmcross/learning-team
+docs/*                                     @mvvmcross/learning-team
+
+MvvmCross/*                                @mvvmcross/maintainers
+MvvmCross/Platforms/Android/*              @mvvmcross/android-team
```
Comment (Cheesebaron) [inline .github/CODEOWNERS]: android

**Comment by Cheesebaron** [comment]
Please remove `-team` suffixes from the teams. Also "learning-team" sounds strange, call it docs please.

**Comment by Cheesebaron** [comment]
The teams have been created 👍

**Pull request merged**